File: tugas1.py
# DONE TUGAS 1
a = lambda start, stop: list(range(start, stop + 1))
print(a(2, 10))

# TUGAS 2 DONE
y = lambda num: "FizzBuzz" if num % 3 == 0 and num % 5 == 0 else "Fizz" if num % 3 == 0 else "Buzz" if num % 5 == 0 else num
x = list(map(y, range(2, 10)))
print(x)

# TUGAS 3 DONE
lambdaa = lambda l: [l[i] + l[i + 1] for i in range(len(l) - 1)]
lis = [1, 2, 3, 4, 5]
p = lambdaa(lis)
print(p)

File: tugas1.dart
List<int> generateRange(int start, int stop) {
  return List<int>.generate(stop - start + 1, (index) => start + index);
}

String fizzBuzz(int num) {
  if (num % 3 == 0 && num % 5 == 0) {
    return "FizzBuzz";
  } else if (num % 3 == 0) {
    return "Fizz";
  } else if (num % 5 == 0) {
    return "Buzz";
  } else {
    return num.toString();
  }
}

List<int> calculateSum(List<int> list) {
  List<int> hasil3 = [];
  for (int i = 0; i < list.length - 1; i++) {
    hasil3.add(list[i] + list[i + 1]);
  }
  return hasil3;
}

void main() {
  List<int> hasil1 = generateRange(2, 10);
  print(hasil1);

  List<String> hasil2 = List.generate(10, (index) => fizzBuzz(index + 2));
  print(hasil2);

  List<int> list1 = [1, 2, 3, 4, 5];
  List<int> hasil3 = calculateSum(list1);
  print(hasil3);
}


